applications/luci-app-statistics/root/etc/config/luci_statistics

```diff
@@ -42,6 +42,11 @@ config statistics 'collectd_logfile'
 config statistics 'collectd_network'
 	option enable '0'
 
+config statistics 'collectd_syslog'
+   option enable '1'
```

Comment:
This should be 0 in the shipped default config, so that there will be no error if there is no plugin.

For that reason only the plugins that are included by default, have 1 as value.